airflow-core/src/airflow/executors/local_executor.py

```diff
@@ -110,6 +110,9 @@ def _execute_work(log: logging.Logger, workload: workloads.ExecuteTask) -> None:
     setproctitle(f"airflow worker -- LocalExecutor: {workload.ti.id}")
 
     base_url = conf.get("api", "base_url", fallback="/")
+    # If it's a relative URL, use localhost:8080 as the default
+    if base_url.startswith("/"):
+        base_url = "http://localhost:8080"
```

Comment:
I thought it should be: 
```suggestion
        base_url = "http://localhost:8080" + base_url 
```

?

Reply:
i think the last part is getting stripped out below always. :)